pyne/mcnp.py

```diff
@@ -2281,7 +2281,7 @@ def _mesh_to_mat_cards(mesh, divs, frac_type):
     mat_cards = ""
     idx = mesh.iter_structured_idx('xyz')
     for i in idx:
-        mesh.mats[i].metadata['mat_number'] = int(i + 1)
-        mat_cards += mesh.mats[i].mcnp(frac_type=frac_type)
+        mesh.mats[int(i)].metadata['mat_number'] = int(i + 1)
```

Comment:
Why add this `int(i)` here? Here, the `i` is a `numpy.int32` type, does it cause an error?